src/v2/view-builder/views/DeviceChallengePollView.js

```diff
@@ -33,6 +34,11 @@ const Body = BaseForm.extend(Object.assign(
       BaseForm.prototype.initialize.apply(this, arguments);
       this.listenTo(this.model, 'error', this.onPollingFail);
       this.deviceChallengePollRemediation = this.options.currentViewState;
+      this.currentRemediationName = this.options.currentViewState.name;
+      this.deviceChallenge = this.deviceChallengePollRemediation.relatesTo.value;
```

Comment:
Does the api's in non-UV flow. ie) just device probing have relatesTo in responses now? 
are the mocks in SIW updated ? Wont this break LEA flows ?

Reply:
Both device probe and user verification has relatesTo, but they are in different format. Thus they need to be handled differently. 
device probe: https://github.com/okta/okta-signin-widget/blob/master/playground/mocks/data/idp/idx/identify-with-device-probing-loopback.json#L13
user verification: https://github.com/okta/okta-signin-widget/pull/1305/files#diff-331e06406f59903637d1ab44fd4e6aa2R14

Reply:
Thanks.